public/scripts/slash-commands.js

```diff
@@ -733,6 +733,47 @@ export function initDefaultSlashCommands() {
         ],
         helpString: 'Unhides a message from the prompt.',
     }));
+    SlashCommandParser.addCommandObject(SlashCommand.fromProps({
+        name: 'member-get',
+        aliases: ['getmember', 'memberget'],
+        callback: (async ({field}, arg) => {
+            if (!selected_group) {
+                toastr.warning('Cannot run /member-get command outside of a group chat.');
+                return '';
+            }
+            if (field === undefined) {
+                throw new Error('\'/member-get field=\' argument required!');
```

Comment:
Personal preference, but I usually prefer toast warnings or errors in slash commands. Also because they allow the execution to continue.
The errors always pop up like an unhandled exception, usually meaning a syntax error or definition issue.
Could be debated if that's the case for missing arguments, but most other commands toast this too.
Error for invalid field identifier is okay.

But that's nitpicky.